cpp/net/url_fetcher.cc

```diff
@@ -276,5 +276,41 @@ ostream& operator<<(ostream& output, const UrlFetcher::Response& resp) {
   return output;
 }
 
+ostream& operator<<(ostream& output, const UrlFetcher::Request& req) {
+  output << "verb: " << req.verb << endl
+         << "url: " << req.url << endl
+         << "headers: " << req.headers << endl
+         << "body: <<EOF" << endl
+         << req.body << "EOF" << endl;
+  return output;
+}
+
+ostream& operator<<(ostream& os, const UrlFetcher::Verb& verb) {
+  switch (verb) {
+    case UrlFetcher::Verb::GET:
+      os << "GET ";
```

Comment:
I don't like those trailing spaces... I can add them when needed, but I removing them would be annoying.